hgeometry/src/Data/Geometry/Polygon/Core.hs

```diff
@@ -116,6 +116,9 @@ _SimplePolygon = prism' SimplePolygon (\(SimplePolygon vs) -> Just vs)
 _MultiPolygon :: Prism' (Polygon Multi p r) (C.CSeq (Point 2 r :+ p), [Polygon Simple p r])
 _MultiPolygon = prism' (uncurry MultiPolygon) (\(MultiPolygon vs hs) -> Just (vs,hs))
 
+instance Functor (Polygon t p) where
+  fmap = bimap id
+
```

Comment:
@noinia Is there a reason why (Polygon t p) shouldn't be a Functor?

Reply:
I explicitly made types that have two type parameters (like polygon, line segment, ext etc) not an instance of functor, since I think that may be somewhat confusing. I.e. that fmap then would manipulate the 'r' argument instead of the 'p' argument. I don't feel super strongly about this though.